stories/calendar/calendar.stories.js

```diff
@@ -341,47 +341,71 @@ export const Months = () => `<div style="display: flex; justify-content: space-a
                 <table class="fd-calendar__table" role="grid">
                     <tbody class="fd-calendar__group">
                         <tr class="fd-calendar__row">
-                            <td role="gridcell" class="fd-calendar__item fd-calendar__item--current fd-button fd-button--transparent">
-                                <span class="fd-calendar__text" role="button">January</span>
+                            <td role="gridcell" class="fd-calendar__item fd-calendar__item--current">
+                                <button class="fd-button fd-button--transparent fd-calendar__item-button" type="button">
```

Comment:
why do you need `type="button"` when you have a native button element?

Reply:
You marked this as resolved but neither changed it nor responded :) 
`<button type="button>` is different from a simple `<button>` with a blank or missing type attribute only in a `form`. There the default type is submit so it makes sense to specify the type, but in the Calendar case I don't see why it's needed.